infra/base-images/base-runner/bad_build_check

```diff
@@ -165,13 +165,32 @@ function check_mixed_sanitizers {
   fi
 }
 
+# Verify that the given fuzz target doesn't crash on the seed corpus.
+function check_seed_corpus {
+  local FUZZER=$1
+  local FUZZER_OUTPUT="/tmp/$(basename $FUZZER).output"
```

Comment:
can you put $(basename $FUZZER) in local FUZZER_NAME or FUZZ_TARGET  and use it in two places in this function.

Reply:
Done.